config/formats.js

```diff
@@ -3420,6 +3426,20 @@ exports.Formats = [
 					};
 				}
 			}
+			if (move.id === 'highjumpkick' && name === 'galbia') {
+				move.name = 'Kibitz';
+				move.basePower = 110;
+				move.accuracy = 100;
+				delete move.onMoveFail;
+				move.onHit = function (target, source) {
+					var result = this.random(100);
+					if (result < (pokemon.ability === 'serenegrace' ? 60 : 30)) {
```

Comment:
`source.hasAbility('serenegrace')`

This lets it work properly with Gastro Acid.